src/MPDevice.cpp

```diff
@@ -1308,7 +1361,14 @@ void MPDevice::startMemMgmtMode(bool wantData,
     jobs->append(startMmmJob);
 
     /* Load flash contents the usual way */
-    memMgmtModeReadFlash(jobs, false, cbProgress, !wantData, wantData, true);
+    if (isBLE())
+    {
+        memMgmtModeReadFlashBLE(jobs, false, cbProgress, !wantData);
```

Comment:
any reason to remove the extra parameters for that function clone? they are needed in the previous gen...

Reply:
Yes, those message mapping havent been implemented for the BLE yet. Thats why a copied only getCredential to a new temporary function (**memMgmtModeReadFlashBLE**). After the commands are mapped it will use `memMgmtModeReadFlash` too just like the mini and I will remove `memMgmtModeReadFlashBLE`.

Reply:
Mentioned in the header: https://github.com/mooltipass/moolticute/pull/440/files#diff-8419d70e7413e49a93c9287a8134a9ecR319

Reply:
ok!